src/page/form/Form.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react'
+
+const subject = [
+  "Đá bóng",
+  "Cầu lông",
+  "Đua thuyền"
+];
+
+export default class Form extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
Bên dưới đang không dùng `props`
nên hãy xóa props đi, vì không cần thiết
```suggestion
  constructor() {
    super();
```